compass.yaml.example

```diff
@@ -65,6 +65,7 @@ worker:
         username: compass
         password: compass_password
     job_manager_port: 8085
+    sync_job_timeout: 15m
```

Comment:
either make it general or make it configurable for each type?

Reply:
IMO configurable each type is preferred, updated